src/stactools/core/utils/antimeridian.py

```diff
@@ -209,10 +215,12 @@ def normalize_multipolygon(multi_polygon: MultiPolygon) -> Optional[MultiPolygon
         Fix this
 
     Args:
-        multi_polygon (:py:class:`shapely.geometry.MultiPolygon`): The input multi-polygon.
+        multi_polygon (:py:class:`shapely.geometry.MultiPolygon`): The input
+            multi-polygon.
 
     Returns:
-        Optional[:py:class:`shapely.geometry.MultiPolygon`]: The normalized multi-polygon.
+        Optional[:py:class:`shapely.geometry.MultiPolygon`]: The normalized
+            multi-polygon.
```

Comment:
This (and other) changes to the `Returns` description does not play well with Sphinx. If the string is multi-line and is indented (which is what it is being changed to here), it does not render correctly in the html docs. Note how the first line is bold, the second normal and indented in the example below.

**_CURRENT HTML DOCS:_**
<img width="750" alt="image" src="https://user-images.githubusercontent.com/7013017/227287775-bb4bfa1c-150a-4d2d-9594-9b4fc839491d.png">

**_HTML DOCS GENERATED FROM THIS PR:_**
<img width="704" alt="image" src="https://user-images.githubusercontent.com/7013017/227287905-9a62efda-34c0-4aa0-b37c-fb48bdbedd74.png">

Reply:
Thanks, good catch. It always weirds me out how indentation works differently for parameters and returns.